### ISSUE TITLE
Tic Watch Pro collection every 5 min ?

### ISSUE THREAD

**Issue by blockrockin** [NightscoutFoundation/xDrip#1399]
Hi , just wondering if there is there any possible chance to create  a patch  for the Tic Watch pro so it could collect a reading from G6 and xdrip every 5 min ?  that would be so great !   thanks

**Comment by Navid200** [comment]
@blockrockin Would you please ask questions from the user community first?
https://www.facebook.com/groups/xDripG5
https://gitter.im/jamorham/xDrip-plus
If you have done that already, please update with your findings.
Thanks

**Comment by blockrockin** [comment]
thanks i did.. from what i could find out it is only possible to collect every 10 min with the tic watch pro as of now ,

**Comment by Navid200** [comment]
Do you think anything can be done from the xDrip side to change that?

**Comment by blockrockin** [comment]
yes i believe their might be  a way to make it happen.  if  someone wanted   to spend time working on that with the xdrip code.? ..   i would love it.   i would like a more updated watch..  but i keep going back to using the sony sw3 because they work so well every 5 min.. .   im keeping my fingers crossed someone might be able to make it happen some day. that would be great !

**Comment by Navid200** [comment]
@blockrockin But, what makes you think this is an xDrip issue and not a watch issue?
You yourself say that your other watch works fine with xDrip.

**Comment by Navid200** [comment]
I'm converting this to a discussion.
